plugins/source/shopify/internal/shopify/shopify.go

```diff
@@ -361,3 +361,7 @@ func getNextPage(hdr http.Header) string {
 
 	return ""
 }
+
+func isErrH2GoAway(err error) bool {
```

Comment:
why not check for `errors.Is(http2.GoAwayError)`?
https://pkg.go.dev/golang.org/x/net/http2#GoAwayError

Reply:
@candiduslynx `http2.GoAwayError` is a type and not a variable, that `errors.Is` won't catch it. [`dfe06d9` (#14273)](https://github.com/cloudquery/cloudquery/pull/14273/commits/dfe06d95b8fe6c1aee925ce64445886d8f8b02cf) uses `errors.As` now, but it isn't as pretty.